packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -0,0 +1,251 @@
+  organizer = 'Organizer',
+}
+
+/**
+ * @hidden
+ * State of the current Live Share sessions backing fluid container.
+ *
+ * @beta
+ */
+export enum ContainerState {
+  /**
+   * The call to `LiveShareHost.setContainerId()` successfully created the container mapping
+   * for the current Live Share session.
+   */
+  added = 'Added',
+
+  /**
+   * A container mapping for the current Live Share Session already exists and should be used
+   * when joining the sessions Fluid container.
+   */
+  alreadyExists = 'AlreadyExists',
+
+  /**
+   * The call to `LiveShareHost.setContainerId()` failed to create the container mapping due to
+   * another client having already set the container ID for the current Live Share session.
+   */
+  conflict = 'Conflict',
+
+  /**
+   * A container mapping for the current Live Share session doesn't exist yet.
+   */
+  notFound = 'NotFound',
+}
+
+/**
+ * @hidden
+ * Returned from `LiveShareHost.get/setFluidContainerId()` to specify the container mapping for the
+ * current Live Share session.
+ *
+ * @beta
+ */
+export interface IFluidContainerInfo {
+  /**
+   * State of the containerId mapping.
+   */
+  containerState: ContainerState;
+
+  /**
+   * ID of the container to join for the meeting. Undefined if the container hasn't been
+   * created yet.
+   */
+  containerId: string | undefined;
+
+  /**
+   * If true, the local client should create the container and then save the created containers
+   * ID to the mapping service.
+   */
+  shouldCreate: boolean;
+
+  /**
+   * If `containerId` is undefined and `shouldCreate` is false, the container isn't ready but
+   * another client is creating it. The local client should wait the specified amount of time and
+   * then ask for the container info again.
+   */
+  retryAfter: number;
+}
+
+/**
+ * @hidden
+ * Returned from `LiveShareHost.getNtpTime()` to specify the global timestamp for the current
+ * Live Share session.
+ *
+ * @beta
+ */
+export interface INtpTimeInfo {
+  /**
+   * ISO 8601 formatted server time. For example: '2019-09-07T15:50-04:00'
+   */
+  ntpTime: string;
+
+  /**
+   * Server time expressed as the number of milliseconds since the ECMAScript epoch.
+   */
+  ntpTimeInUTC: number;
+}
+
+/**
+ * @hidden
+ * Returned from `LiveShareHost.getFluidTenantInfo()` to specify the Fluid service to use for the
+ * current Live Share session.
+ *
+ * @beta
+ */
+export interface IFluidTenantInfo {
+  /**
+   * The Fluid Tenant ID Live Share should use.
+   */
+  tenantId: string;
+
+  /**
+   * The Fluid service endpoint Live Share should use.
+   */
+  serviceEndpoint?: string;
+
+  /**
+   * @deprecated
+   * As of Fluid 1.0 this configuration information has been deprecated in favor of
+   * `serviceEndpoint`.
+   */
+  ordererEndpoint: string;
+
+  /**
+   * @deprecated
+   * As of Fluid 1.0 this configuration information has been deprecated in favor of
+   * `serviceEndpoint`.
+   */
+  storageEndpoint: string;
+}
+
+/**
```

Comment:
Probably want @hidden on the class as well

Reply:
Talked with Steve, the class and the create function actually do want to show in the docs for developers.